cli/src/main/scala/filodb.cli/CliMain.scala

```diff
@@ -191,6 +196,21 @@ object CliMain extends ArgMain[Arguments] with CsvImportExport with FilodbCluste
               println("Either --filename or --configPath must be specified for setup")
               exitCode = 1
           }
+        case Some("timeseriesMetadata") =>
+          require(args.host.nonEmpty && args.dataset.nonEmpty && args.matcher.nonEmpty, "--host, --dataset and --matcher must be defined")
+          val remote = Client.standaloneClient(system, args.host.get, args.port)
+          val options = QOptions(args.limit, args.sampleLimit, args.everyNSeconds.map(_.toInt),
+            timeout, args.shards.map(_.map(_.toInt)), spread)
+          parseTimeSeriesMetadataQuery(remote, args.matcher.get, args.dataset.get,
+            new TimeStepParams(args.start, -1, args.end), options)
```

Comment:
Can we please use `getQueryRange` method here?  That way we preserve the ability to use convenient options I just added like --minutes and other debugging options

Reply:
@velvia ack